src/Service.Host/client/src/components/parts/tabs/PurchTab.js

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import { InputField, Dropdown, Typeahead, LinkButton } from '@linn-it/linn-form-components-library';
+
+function PurchTab({
+    handleFieldChange,
+    unitsOfMeasure,
+    ourUnitOfMeasure,
+    preferredSupplier,
+    handlePrefferedSupplierChange,
+    preferredSupplierName,
+    suppliersSearchResults,
+    suppliersSearchLoading,
+    searchSuppliers,
+    clearSuppliersSearch,
+    currency,
+    currencyUnitPrice,
+    baseUnitPrice,
+    materialPrice,
+    labourPrice,
+    costingPrice,
+    orderHold,
+    partCategory,
+    nonForecastRequirement,
+    oneOffRequirement,
+    sparesRequirement,
+    ignoreWorkstationStock,
+    handleIgnoreWorkstationStockChange,
+    imdsIdNumber,
+    imdsWeight,
+    mechanicalOrElectronic,
+    partCategories
+}) {
+    const convertToYOrNString = booleanValue => {
```

Comment:
Not sure if it's worth it but if this is wee func is used in a enough places would there be a benefit to making it somewhere centralised/its own component?